test/test-fs.c

```diff
@@ -1290,7 +1291,10 @@ TEST_IMPL(fs_mkstemp) {
   ASSERT(strcmp(mkstemp_req1.path, mkstemp_req2.path) != 0);
 
   /* invalid template returns EINVAL */
-  ASSERT(uv_fs_mkstemp(NULL, &mkstemp_req3, "test_file", NULL) == UV_EINVAL);
+  ASSERT(uv_fs_mkstemp(NULL, &mkstemp_req3, test_file, NULL) == UV_EINVAL);
+
+  /* Make sure that path is empty string */
+  ASSERT(strlen(mkstemp_req3.path) == 0);
```

Comment:
Can you use the new-style macros?
```suggestion
  ASSERT_EQ(UV_EINVAL, uv_fs_mkstemp(NULL, &mkstemp_req3, test_file, NULL));

  /* Make sure that path is empty string */
  ASSERT_EQ(0, strlen(mkstemp_req3.path));
```

Reply:
Moved to new macros.